src/App.tsx

```diff
@@ -29,7 +23,7 @@ function App() {
   };
 
   //Handle search filtering of tasks
-  const filteredTasks = tasks.filter((task) => {
+  const filteredTasks = tasksData.filter((task) => {
     if (searchInput === '') {
       return task;
     }
```

Comment:
🪡 Nitpick: you could move this into your hook

Reply:
moved to `useTaskManager.ts` hook